app/models/label/LabelTable.scala

```diff
@@ -728,9 +729,21 @@ object LabelTable {
     // Init random function.
     val rand = SimpleFunction.nullary[Double]("random")
 
+    // Prioritize labels to show by filtering out those that have been
```

Comment:
Reminder that max line length is 120 characters (also period at the end of comment).

Reply:
yup

Reply:
What about lines like in the inner joins with a ton of join conditions?

Reply:
I was actually noting the line length because you aren't filling up the 120 characters, and the comment could be cut down to 2 lines from 3 if the full width was used.

But for extra long lines, I think that there are some exceptions. The one you suggested is a decent exception sometimes. Usually a variable can be renamed or something to shorten the line enough. In the case below, I actually think that `_lb.labelTypeId === labelTypeId && _lb.streetEdgeId =!= tutorialStreetId` doesn't need to be on that line. I actually usually prefer to have the filtering down at the bottom (to mimic how the `WHERE` clause is usually at the end). Unfortunately, I think that a weird quirk of Slick is that wherever we use `inSet` that can't be tacked on at the end for some reason.